.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env"]
```

Comment:
This would default to CJS module output no? If so, should we be switching this to "modules":false?

Reply:
The client source is still using `require`, so having `modules: false` has no effect. Or we can convert to use ES modules imports and enable `modules: false`, but that may break compat with webpack 2 (not sure if that's still a requirement)